inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/metrics/stats/MonitorIndex.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.dataproxy.metrics.stats;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.concurrent.atomic.LongAdder;
+
+public class MonitorIndex extends AbsStatsDaemon {
+
+    private static final Logger logger = LoggerFactory.getLogger(MonitorIndex.class);
+
+    private static final AtomicLong recId = new AtomicLong(0);
+    private final StatsUnit[] statsUnits = new StatsUnit[2];
+
+    public MonitorIndex(String name, long intervalMill, int maxCnt) {
+        super(name, intervalMill, maxCnt);
+        this.statsUnits[0] = new StatsUnit(name);
+        this.statsUnits[1] = new StatsUnit(name);
+    }
+
+    /**
+     * addSuccStats
+     *
+     * @param key
```

Comment:
As we can see, if there is no explanation after the param parameter in the javadoc, there will be a warning message in some IDEs; in addition, the javadoc without description has no practical meaning.

So, please add description information for these parameters, or delete the declaration of these params in javadoc.

![image](https://github.com/apache/inlong/assets/31994335/bf00cf32-28b3-4e1b-add8-cb5cae3c54bf)